daemon/stats/collector_unix.go

```diff
@@ -8,14 +8,17 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/opencontainers/runc/libcontainer/system"
 	"golang.org/x/sys/unix"
 )
 
 // platformNewStatsCollector performs platform specific initialisation of the
 // Collector structure.
 func platformNewStatsCollector(s *Collector) {
-	s.clockTicksPerSecond = uint64(system.GetClockTicks())
+	// The value comes from `C.sysconf(C._SC_CLK_TCK)`, and
+	// on Linux it's a constant which is safe to be hard coded,
+	// so we can avoid using cgo here. For details, see:
+	// https://github.com/containerd/cgroups/pull/12
+	s.clockTicksPerSecond = uint64(100)
```

Comment:
Perhaps this should now just be a `const clockTicksPerSecond` (grouped with the `nanoSecondsPerSecond` below), and then we can remove the `clockTicksPerSecond` property, and `platformNewStatsCollector()`.

I think this change would make sense as a separate commit (or even a separate PR)